testing/config/settings/base.py

```diff
@@ -122,3 +122,21 @@
         }
     }
 }
+
+"""
+Settings for improved performance, when testing
```

Comment:
I would prefer to put these settings in a dedicated "testing" settings file. But the way the settings are structured now, this was not easily possible. So I put those at the end of base.py.

What do you think?